enterprise/views.py

```diff
@@ -438,18 +440,23 @@ def post_course_specific_consent(self, request, course_id, consent_provided):
         except HttpClientError:
             raise Http404
 
-        enrollment_deferred = request.POST.get('enrollment_deferred')
```

Comment:
when I was going through this to update the tests, i was confused about the purpose of this enrollment_deferred variable as well as the consequences of removing it.... can you explain?